scripts/verify_ledger_json.js

```diff
@@ -1,31 +1,34 @@
+'use strict';
+
 var fs = require('fs');
+var Amount = require('../src/js/ripple').Amount;
 var Ledger = require('../src/js/ripple/ledger').Ledger;
 
 function parse_options(from, flags) {
   var argv = from.slice(),
-      opts = {argv:argv};
+      opts_ = {argv: argv};
 
   flags.forEach(function(f) {
     // Do we have the flag?
     var flag_index = argv.indexOf('--' + f);
     // normalize the name of the flag
     f = f.replace('-', '_');
-    // opts has Boolean value for normalized flag key
-    opts[f] = !!~flag_index;
-    if (opts[f]) {
+    // opts_ has Boolean value for normalized flag key
+    opts_[f] = !!~flag_index;
```

Comment:
I know this isn't part of your pull request, but we should try to replace `!!~x` with `x !== -1` or use `_.includes` instead. You don't have to do it now, but I just want to point out these cases.